modules/ndu/src/backend/migrate.ts

```diff
@@ -107,11 +107,12 @@ const createMissingElements = async (bp: typeof sdk, botId, files: TemplateFile[
 }
 
 const getTemplateFiles = async (bp: typeof sdk): Promise<TemplateFile[]> => {
-  return (await bp.bots.getBotTemplate('ndu', 'oneflow')).map(x => ({
-    fileName: x.name.replace(/\\/g, '/'),
-    content: JSON.parse(x.content.toString()),
-    buffer: x.content
-  }))
+  return []
+  // return (await bp.bots.getBotTemplate('ndu', 'oneflow')).map(x => ({
+  //   fileName: x.name.replace(/\\/g, '/'),
+  //   content: JSON.parse(x.content.toString()),
+  //   buffer: x.content
+  // }))
```

Comment:
I have no idea what we want or should do here, but in any case, either uncomment the code or remove it.

Reply:
NDU is going to be phased out eventually, I just commented the code so we can have a quick look at it, but in the best case we'll just delete the module completely

Reply:
Then can you add a comment/TODO!